v4/export/config.go

```diff
@@ -242,7 +244,7 @@ func (conf *Config) DefineFlags(flags *pflag.FlagSet) {
 	flags.StringToString(flagParams, nil, `Extra session variables used while dumping, accepted format: --params "character_set_client=latin1,character_set_connection=latin1"`)
 	flags.Bool(FlagHelp, false, "Print help message and quit")
 	flags.Duration(flagReadTimeout, 15*time.Minute, "I/O read timeout for db connection.")
-	_ = flags.MarkHidden(flagReadTimeout)
+	flags.Duration(flagMaxIdleTime, 600*time.Second, "max idle time for db connection.")
```

Comment:
We'd better keep `flagReadTimeout` hidden. What's more, we'd better keep `flagMaxIdleTime` hidden, too. We can "un-hidden" these arguments if they are frequently used.

Reply:
We can add these flags to official documents.